domain_config_ui/domain_config_ui.services.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  domain_config_ui.manager:
+    class: Drupal\domain_config_ui\DomainConfigUIManager
+    arguments: ['@config.storage', '@domain.loader', '@language_manager']
```

Comment:
The reason was to keep the loading of config that's being edited as self-contained as possible.

Spoofing the requested domain for the negotiator meant that all the config on the page also got swapped. It didn't seem right to have an admin tool overriding the negotiator and change its behaviour.

I'm also not sure if we want setting overrides to be applied to editable values?

I can change this back if you think swapping all config on the page is better?